portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -126,7 +85,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         // Get the input from the form.
-        /*System.out.println("test");
+        System.out.println("test");
```

Comment:
Nit: would be good to give debug output a bit more concrete context, e.g., which class/function, etc. That will be super useful in large scale code:) Otherwise, you will be seeing a lot of "test", which you have no clue where it is.